### PR TITLE
[FEATURE] Added functionality for calculating gross prices

### PR DESCRIPTION
**Pull request by turbo-ele**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     "description": "TODO",
     "license": "MIT",
     "keywords": ["TODO"],
+    "version": "0.1",
```
Comment (turbo-ele) [inline composer.json]: remove

File: Pricing/ItemPriceCalculator.php [hunk 2]
```diff
@@ -201,11 +209,17 @@ private function getValidProductPriceForItem($item, $currency)
...
 
-        // no price set - return 0
+        // If no price is set - return 0.
         if (empty($priceValue)) {
             return 0;
         }
 
+        // Check if product price is gross price and return net price instead.
+        if ($product->getAreGrossPrices()) {
+            // FIXME: getTax not in interface. Needs to be fetched from product instead.
```
Comment (turbo-ele) [inline Pricing/ItemPriceCalculator.php]: fetch from product

File: Pricing/PriceCalculationManager.php [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+    }
+
+    /**
+     * Unsets all data from array that's not needed for price calculation.
+     *
+     * @param array $data
+     *
+     * @return array
+     */
+    private function unsetUneccesaryData($data)
```
Comment (christianbader) [inline Pricing/PriceCalculationManager.php]: type

File: Pricing/PriceCalculationManager.php [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    }
+
+    /**
+     * Sets default data to data array, if needed for price calculation.
+     *
+     * @param array $data
+     *
+     * @return array
+     */
+    private function setDefaultData($data)
```
Comment (christianbader) [inline Pricing/PriceCalculationManager.php]: type

**Comment by christianbader** [comment]
ack

**Pull request merged**